src/test/java/org/opensearch/flowframework/transport/CreateWorkflowTransportActionTests.java

```diff
@@ -51,18 +52,24 @@ public class CreateWorkflowTransportActionTests extends OpenSearchTestCase {
     private ThreadPool threadPool;
     private ParseUtils parseUtils;
     private ThreadContext threadContext;
+    private Settings settings;
 
     @Override
     public void setUp() throws Exception {
         super.setUp();
         threadPool = mock(ThreadPool.class);
+        settings = Settings.builder()
+            .put("plugins.flow_framework.max_workflows.", 2)
+            .put("plugins.anomaly_detection.backoff_initial_delay", TimeValue.timeValueSeconds(10))
```

Comment:
I think this should be `plugins.flow_framework.request_timeout`

Reply:
It's a random setting value and can be treated as cluster settings . I can change it to the above.